resources/js/login/login.js

```diff
@@ -717,6 +812,11 @@ class LoginPage extends React.Component {
                                 <OTPHelpLinks emitOtpAction={this.handleEmitOtpAction}/>
                             </>
                         }
+                        <CustomSnackbar
```

Comment:
@romanetar this is rendered always ... should not be conditionally rendered if this.state.notification.message !== ?